internal/string.go

```diff
@@ -72,3 +72,7 @@ func IntsToStrings(numbers []int) []string {
 	}
 	return strs
 }
+
+func CopyStringSlice(src []string) []string {
+	return append(make([]string, 0), src...)
```

Comment:
What reallocation could be prevented? In case of len(src)==0 we cannot return src - we still need to make/allocate empty  slice to be it separate copy and not reference to the same empty slice.